index.html

```diff
@@ -12,10 +12,103 @@
+        <img id="header-img" class="logo" src="karma-logo.svg"  alt= "Karma's logo, Orange background with white element"><!-- Karma logo-->
+        <nav id="nav-bar"><!-- Navigation bar -->
+            <ul><!-- elements on navigation bar -->
+                <li><a href="#meetKarma"> Meet Karma </a></li>
+                <li><a href="#howItWorks"> How It Works</a></li>
+                <li><a href="#store">Store</a></li>
+                <li><a href="#blog">Blog</a></li>
+                <li><a href="#help">Help</a></li>
+                <li><a href="#login">Login</a></li>
+            </ul>
+            </nav>
+    </header>
+    <main>
+        <section>
+            <img id="bg-img" src="img/first-background.jpg" alt="Someone's half body and a tablet on table">
+    </section>
+    <section id="intro-heading">
+        <p>Introducing Karma</p>
+    </section>
+    <section id="intro-text">
+        <p>Bring WiFi with you, everywhere you go.</p>
+    </section>
+    <section> 
+        <button class="intro-button">Learn More</button>
+    </section> 
+
+    <section>
+        <div id="container-heading">
+                <p>Every one needs a little Karma</p>
+        </div>
+
+  <div id="container-items">
+          <section>
+            <img
+              class="container-icons"
+              src="img/icon-devices.svg"
+              alt="Computer, tablet and cellphone icon"
+            />
+            <p class="container-text">Internet for all devices</p>
+          </section>
+          <section>
+            <img
+              class="container-icons"
+              src="img/icon-coffee.svg"
+              alt="cup of coffee icon"
+            />
+            <p class="container-text">Boost your productivity</p>
+          </section>
+          <section>
+            <img
+              class="container-icons"
+              src="img/icon-refill.svg"
+              alt="gass container logo"
+            />
+            <p class="container-text">Pay as you go</p>
+          </section>
+        </div>
 
+    </section>
+    </main>
+    <hr>
+    <footer>
+        <section>
+             <div id="footer-heading">
+                 <p>Join us on</p>
+            </div>
+             <div id="footer-item">
+          <section>
+            <img
+              class="footer-icons"
+              src="img/twitter-icon.svg"
+              alt="twitter-logo"
+            />
+          </section>
+          <section>
+            <img
+              class="footer-icons"
+              src="img/facebook-icon.svg"
+              alt="facebook-logo"
+            />
+          </section>
+          <section>
+            <img
+              class="footer-icons"
+              src="img/instagram-icon.svg"
+              alt="instagram-logo"
+            />
+          </section>
+        </div>
+    </section>
+    <section class="footer-copyright">
+        <p> &copy Karma Mobility,Inc. </p>
+    </section>
+    </footer>
 </body>
 </html>
```

Comment:
Overall, very good use of semantic tags such as `<main>`, `<section>`, `<footer>` etc.